core/transcoder/transcoder.go

```diff
@@ -447,8 +447,3 @@ func (t *Transcoder) SetIdentifier(output string) {
 func (t *Transcoder) SetInternalHTTPPort(port string) {
 	t.internalListenerPort = port
 }
-
-// SetCodec will set the codec to be used for the transocder.
-func (t *Transcoder) SetCodec(codecName string) {
```

Comment:
Why was the setter removed? If anything I think we should standardize on setters instead of setting properties directly.

Reply:
Yes, I agree.